nav2_docking/opennav_docking/test/test_docking_server.py

```diff
@@ -172,7 +190,7 @@ def test_docking_server(self) -> None:
         self.tf_broadcaster = TransformBroadcaster(self.node)
 
         # Create a timer to run "control loop" at 20hz
-        self.timer = self.node.create_timer(0.05, self.timer_callback)
+        self.timer = self.node.create_timer(0.033, self.timer_callback)
```

Comment:
Why change the timer and the `self.period`? Just curious.

Reply:
Adding two more tests was computationally expensive, so I tried to reduce the duration of the tests.

Reply:
If we keep this, the comment on the line above should change from 20hz->30hz. Even better, we should probably replace 0.05 or 0.033 with the new self.period (and maybe just remove the "at 20hz" part from this comment since the actual rate is defined elsewhere when creating self.period)